### DISCUSSION TITLE
Updated TODO list and contribution guidelines

### DISCUSSION THREAD

**Discussion by MatteoGheza** [josephdadams/TallyArbiter#766] [General]
Hello,
I’ve been away from the project for quite some time due to other commitments. I’d now like to focus on addressing bugs introduced after the v3 release (I see there are lots of new bug reports) and also look into merging some older PRs, such as #358.
Do you have any suggestions on how to proceed, @josephdadams?

**Comment by josephdadams** [comment]
@MatteoGheza go for it! I haven’t had as much time lately.
